Tests/Unit/LoRaTools/RegionAS923TestData.cs

```diff
@@ -59,7 +67,7 @@ public static IEnumerable<object[]> TestRegionFrequencyData()
            {
                new object[] { region, 1, 8 },
                new object[] { region, 1, 9 },
-               new object[] { regionWithDwellTime, 1, 10 },
+               new object[] { regionWithDwellTime, 2, 10 },
```

Comment:
Just curious, was this test case invalid? Similar for deleted lines 46-47.

Reply:
Nice catch. My original reasoning was that since for downstream data rates, due to dwell time limits enforce DR2 and higher that this would automatically apply for upstream data rates, since dwell time limits both upstream and downstream. Since this is not specified explicitly, I reverted that assumption in fb5d856e5c1bca8aad8d8fd90102f14fbfb7d1ce, but I forgot to bring back the original test cases. Solved this in 033e20937e3c79c335e5bd7c7fd83f6f24c26435